includes/class-newspack-newsletters-layouts.php

```diff
@@ -120,12 +120,35 @@ public static function register_meta() {
 	 * @return string Content.
 	 */
 	public static function layout_token_replacement( $content, $extra = [] ) {
+		$home_url       = get_home_url();
 		$sitename       = get_bloginfo( 'name' );
 		$custom_logo_id = get_theme_mod( 'custom_logo' );
 		$logo           = $custom_logo_id ? wp_get_attachment_image_src( $custom_logo_id, 'medium' )[0] : null;
+		$date           = gmdate( get_option( 'date_format' ) );
+		$bg_color       = '#ffffff';
+		$text_color     = '#000000';
+
+		if ( function_exists( 'newspack_setup' ) ) {
```

Comment:
Does that serve the purpose of determining if a Newspack theme is in use? If so, that's not readily understandable, there should be a comment here.

Reply:
fixed: https://github.com/Automattic/newspack-newsletters/pull/709/commits/e3fd6558b5107efc475d99bcc9688ce8165a0fd8